crates/server/src/datastore/postgres/mod.rs

```diff
@@ -251,12 +251,12 @@ where
             })?;
 
             // Validate the record
-            validator.validate(&record).map_err(Into::into)?;
+            let validator = validator.0.validate(&record).map_err(Into::into)?;
```

Comment:
Note that I didn't bother renaming "validator" here as the column name is still "validator" and not "state"; I figured it wasn't worth a migration.